src/content/docs/codestream/codestream-settings/team-administration.mdx

```diff
@@ -1,64 +1,67 @@
+
+![A screenshot showing the organization member list](./images/MyOrgMembers.png "organization-members")
 
-![This image shows the admin organization view.](/images/AdminRights2.png "Admin organization view")
+### Invite your teammates [#invite]
 
-<figcaption>Use the organization menu to control access and permissions.</figcaption>
+Click **Invite Teammates** to invite new members to your organization, see outstanding invites, and suggested teammates.
 
-## Use blame map to reassign code [#blame-map]
+  * The outstanding invitations section lists all open invitations. The right side of each row has links to remove the invitation or to resend the invite.
+  * Click **reinvite** to send another invitation via email. You can also hover over the reinvite link for the option to generate an email yourself.
+  * The suggested teammates section, only available for admins, is a list of possible teammates derived from the commit history of your open repositories. At the right side of each row are links to remove the suggestion from the list or to invite the person.
 
-Click **Blame Map** to define code ownership with your organization.
+### Blame map [#blame]
 
-By default, when you comment on code, CodeStream mentions (or offers to email) the author or authors of the code you're commenting on. If that person has left the company it might not be the right thing to do. 
+Click **Blame Map** to add email addresses that you use for committing code that may be different from the email address you used to sign up for CodeStream. For example, your CodeStream email address might be dave@acme.com, but you might also commit code as `dave@webmail.com`. 
 
-Non-admins are able to set up blame maps for themselves to handle situations where the email address they use to commit code is different than the one they used to sign up for CodeStream.
+Click **Add mapping**, enter your Git email address, and then select your entry from the list of organization members. That way, when someone comments on code committed by dave@webmail.com, CodeStream will know to at-mention you (such as, `dave@acme.com`).
+
+![A screenshot showing a blame map.](./images/MyOrgBlameMap.png "blame-map")
+
+While non-admins can only create blame maps for themselves, admins can create blame maps for any member of the organization. This is useful for reassigning code ownership when people leave the organization.
 
 ## Organization settings [#organization-settings]
 
-If you're an organization admin, look for the **Organization Admin** menu under the headshot menu at the top of the CodeStream pane.
+If you're an admin, use the **Organization Admin** menu under the headshot menu at the top of the CodeStream pane to change your organization's name, manage [onboarding](#onboarding) and [feedback request](#feedback-request) settings, [export data](#export), or delete your org.
```

Comment:
The "feedback request" link doesn't appear to be working. I think the anchor link is missing an "s" at the end.

Reply:
Good catch, thanks!